pygeoapi/provider/mvt_tippecanoe.py

```diff
@@ -253,21 +281,20 @@ def get_html_metadata(self, dataset, server_url, layer, tileset,
         metadata['tileset'] = tileset
         metadata['collections_path'] = service_url
         metadata['json_url'] = f'{metadata_url}?f=json'
-        # Some providers may not implement tilejson metadata
-        metadata['tilejson_url'] = f'{metadata_url}?f=tilejson'
-
-        if not isinstance(self.service_metadata_url, Path):
-            msg = f'Wrong data path configuration: {self.service_metadata_url}'  # noqa
-            LOGGER.warning(msg)
-        elif self.service_metadata_url.exists():
-            with open(self.service_metadata_url, 'r') as md_file:
-                metadata_json_content = json.loads(md_file.read())
-                if 'metadata_json_content' in locals():
-                    content = MVTTilesJson(**metadata_json_content)
-                    content.tiles = service_url
-                    content.vector_layers = json.loads(
-                            metadata_json_content["json"])["vector_layers"]
-                    metadata['metadata'] = content.model_dump()
+
+        try:
+            metadata_json_content = self.get_metadata_from_URL(self.service_metadata_url) # noqa
+
+            content = MVTTilesJson(**metadata_json_content)
+            content.tiles = service_url
+            content.vector_layers = json.loads(
+                    metadata_json_content["json"])["vector_layers"]
+            metadata['metadata'] = content.model_dump()
+            # Some providers may not implement tilejson metadata
+            metadata['tilejson_url'] = f'{metadata_url}?f=tilejson'
+        except ProviderConnectionError:
+            # No vendor metadata JSON
+            pass
```

Comment:
@PascalLike what about to log and raise the `ProviderConnectionError` exception rather than pass? Something like

```python
except ProviderConnectionError as e:
    logger.debug(e)
    raise
```

Reply:
It raises the exception if tilejson is required. To raise the exception in this point too will block the functioning of the html page, which otherwise has the link to the metadata in OGC Tile Set Metadata format and the other primary information, which is the only very mandatory information. But I was undecided too. Do you think it's better to raise the exception here too? Thanks